test/vector_tile_rasterize.cpp

```diff
@@ -168,9 +168,9 @@ TEST_CASE("vector tile rasterize -- should try to decode windfail tile")
         mapnik::agg_renderer<mapnik::image_rgba8> ren(map,im);
         ren.apply();
         unsigned diff = testing::compare_images(im,"test/fixtures/rasterize-expected-1.png");
-        // should be almost equal (50 is good enough since re-rendering filters a few small degenerates)
-        CHECK(50 == diff);
-        if (diff > 50)
+        // should be almost equal (110 is good enough since re-rendering filters a few small degenerates)
+        CHECK(110 == diff);
+        if (diff > 110)
         {
```

Comment:
@flippmoke no, its not. It is static and only generated if it does not exist. So I re-created it just now to see if the diff would change. It did change and so I've checked in the new image: https://github.com/mapbox/mapnik-vector-tile/pull/204/commits/34d1b8e3311efbc432f89abc2a7d8de38c2dd6d6